Library/Homebrew/sandbox.rb

```diff
@@ -8,18 +8,6 @@ def self.available?
     OS.mac? && File.executable?(SANDBOX_EXEC)
   end
 
-  # there are times the sandbox cannot be used.
-  def self.auto_disable?
-    @auto_disable ||= ARGV.interactive? || ARGV.debug?
-  end
-
-  def self.print_autodisable_warning
-    unless @printed_autodisable_warning
-      opoo "The sandbox cannot be used in debug or interactive mode."
-      @printed_autodisable_warning = true
-    end
-  end
```

Comment:
The reason it used to be disabled is that it's hard to retain file in interactive shell. So we are safely to remove them for now.